.github/workflows/build.yaml

```diff
@@ -562,6 +567,11 @@ jobs:
           docker pull ${{ env.REGISTRY }}/${{ env.NAMESPACE }}/galasa-mvp:main
           docker save -o ${{ github.workspace }}/isolated/mvp/target/isolated/isolated.tar ${{ env.REGISTRY }}/${{ env.NAMESPACE }}/galasa-mvp:main
```

Comment:
The docs .zip needs to go into the isolated zip also.

Reply:
Thank you, I have added this in https://github.com/galasa-dev/isolated/pull/115/commits/1ef179bfa5b842da93890986c32045e00539d0b7